lighthouse.go

```diff
@@ -1170,6 +1170,9 @@ func (lhh *LightHouseHandler) handleHostUpdateNotification(n *NebulaMeta, vpnAdd
 		useVersion = 2
 	}
 
+	//todo hosts with only v2 certs cannot provide their ipv6 addr when contacting the lighthouse via v4?
+	//todo why do we care about the vpnip in the packet? We know where it came from, right?
```

Comment:
Its a great question and I have it in my internals notes as well as ditching the V4AddrPort in v2 protocol stuff.